File: Exam.java
import java.util.*;

public class Exam{
	public static void main(String args[]){
		Scanner sc = new Scanner(System.in);
		int n = sc.nextInt();
		String myanswers = sc.next();
		String friendanswers = sc.next();
		int same = 0;
		int dif = 0;

		for(int i = 0; i < myanswers.length(); i++){
			if(friendanswers.charAt(i) == myanswers.charAt(i)){
				same++;
			}else{
				dif++;
			}
		}

		int ans = 0;

		if(n < same){
			ans = n + dif;
		}else{
			ans = dif - n;
		}

		System.out.println(ans);

	}
}

File: Exam.cpp
#include <iostream>

using namespace std;

int main(){
	int n;

	cin >> n;

	string myfriend, me;

	cin >> me >> myfriend;
	int dif = 0;
	int same = 0;

	for(int i = 0; i < me.size(); i++){
		if(me[i] == myfriend[i]){
			same++;
		}else{
			dif++;
		}
	}

	int ans = 0;

	if (n > same)
	{
		ans = me.size() - (n - same);
	}else{
		ans = dif + n;
	}

	cout << ans << endl;
}